grammars/src/main/antlr/BaseLexer.g4

```diff
@@ -151,8 +151,9 @@ fragment LABEL : ALPHA_CHAR (NAME_CHAR|URI_PCT_ENCODED)* ;
 
 GUID : HEX_DIGIT+ '-' HEX_DIGIT+ '-' HEX_DIGIT+ '-' HEX_DIGIT+ '-' HEX_DIGIT+ ;
 
-ALPHA_UC_ID : ALPHA_UCHAR WORD_CHAR* ;           // used for type ids
-ALPHA_LC_ID : ALPHA_LCHAR WORD_CHAR* ;           // used for attribute / method ids
+ALPHA_UC_ID :   ALPHA_UCHAR WORD_CHAR* ;   // used for type ids
+ALPHA_LC_ID :   ALPHA_LCHAR WORD_CHAR* ;   // used for attribute / method ids
+ALPHA_UNDERSCORE_ID : '_' WORD_CHAR* ;     // usually used for meta-model ids
```

Comment:
```suggestion
UNDERSCORE_ID : '_' WORD_CHAR* ;     // usually used for meta-model ids
```
As the format seems to be `firstchar_ID`.

Reply:
I copied this from changes in the 'official' grammar at https://github.com/openEHR/adl-antlr/blob/master/src/main/antlr/adl2/base_lexer.g4#L150 , and I would like to keep this aligned as much as possible.

Reply:
This is just changing the lexer rule id? Not sure I see the value - LC_ID = 'lower case id', UC_ID = 'upper case id'; 'UNDERSCORE_ID' = underscore id. If we want nicer names I'd rather do that in the major revamp I am working on.

Reply:
> I copied this from changes in the 'official' grammar at https://github.com/openEHR/adl-antlr/blob/master/src/main/antlr/adl2/base_lexer.g4#L150 , and I would like to keep this aligned as much as possible.

Agreed.

> This is just changing the lexer rule id? Not sure I see the value - LC_ID = 'lower case id', UC_ID = 'upper case id'; 'UNDERSCORE_ID' = underscore id. If we want nicer names I'd rather do that in the major revamp I am working on.

I interpreted the rule names as: "ALPHA_UC_ID": ID starting with an ALPHA_UC (upper case letter), "ALPHA_LC_ID": ID starting with an ALPHA_LC (lower case letter).

Then "ALPHA_UNDERSCORE_ID" would be ID starting with an ALPHA_UNDERSCORE, but that is strange. For me, "UNDERSCORE_ID" makes more sense: ID starting with an UNDERSCORE.

But if it is already in use, maybe changing it isn't worth it.